package.json

```diff
@@ -6,28 +6,37 @@
   "private": true,
   "scripts": {
     "build": "rollup -c",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:ci": "jest --ci",
+    "lint": "foundry run eslint . --ext .js,.jsx,.json,.ts,.tsx",
+    "lint:fix": "yarn lint --fix",
+    "lint:ci": "yarn lint --format junit -o __reports__/eslint-results.xml"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.12.0",
-    "@typescript-eslint/parser": "^1.12.0",
+    "@sumup/foundry": "^3.1.1",
+    "@types/jest": "^25.1.3",
+    "@types/tti-polyfill": "^0.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.6.1",
+    "@typescript-eslint/parser": "^4.6.1",
     "cz-conventional-changelog": "3.0.2",
-    "eslint": "^5.0.0",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-prettier": "^3.1.0",
-    "prettier": "^1.18.2",
-    "rollup": "^1.19.3",
-    "rollup-plugin-commonjs": "^10.0.2",
-    "rollup-plugin-json": "^4.0.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-polyfill": "^3.0.0",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-terser": "^5.1.1",
-    "rollup-plugin-typescript2": "^0.22.1",
+    "eslint-config-airbnb-typescript": "^12.0.0",
+    "jest": "^25.1.0",
+    "ts-jest": "^25.2.1",
     "typescript": "^3.5.3"
   },
   "dependencies": {
-    "dotenv": "^8.1.0",
+    "@rollup/plugin-commonjs": "^16.0.0",
+    "@rollup/plugin-json": "^4.1.0",
+    "@rollup/plugin-node-resolve": "^10.0.0",
+    "@rollup/plugin-replace": "^2.3.4",
+    "@rollup/plugin-strip": "^2.0.0",
+    "@rollup/plugin-typescript": "^6.1.0",
+    "@sumup/performance-observer": "^1.0.2",
+    "dotenv": "^8.2.0",
+    "rollup": "^2.33.1",
+    "rollup-plugin-terser": "^7.0.2",
```

Comment:
Can you please explain why the Rollup dependencies aren't `devDependencies`? Is it because they're used when generating a custom script?

Reply:
Correct